deploy.sh

```diff
@@ -57,6 +58,7 @@ elif [ "$1" == "gtest-setup" ]; then
 	sudo apt-get update
 	sudo apt-get install -y build-essential
 	sudo apt-get install -y cmake libgtest-dev
+    sudo apt-get install -y python3 python3-pip python-is-python3 clang-format
```

Comment:
Want to move this line to a separate "setup" argument. Since we don't want to install gtest on our beaglebones having our "gtest-setup" and "setup" functions separated would allow us to do this